modules/luci-base/luasrc/cbi/datatypes.lua

```diff
@@ -268,17 +277,24 @@ function wepkey(val)
 end
 
 function hexstring(val)
-        if val then
-                return (val:match("^[a-fA-F0-9]+$") ~= nil)
-        end
-        return false
+	if val then
+		return (val:match("^[a-fA-F0-9]+$") ~= nil)
+	end
+	return false
+end
```

Comment:
spaces to tabs, sorry. Didn't mean to touch your code. At least now it is consistent :)